### PR TITLE
fix(configure): allow short username

### LINKED ISSUE #22: Username validity check incorrect
The `swan configure` username prompt rejects my 4-character username, because... reasons?
See [`bin/swan-configure.js:18`](https://github.com/SpoonX/swan-cli/blob/63873c517cc6f5d7cd0e88b408d1a95370c82284/bin/swan-configure.js#L18).

### PR DESCRIPTION
**Pull request by ppvg**
Allow any username with at least one non-whitespace character (really just a simple check to disregard accidental whitespace).

Fixes #22.

### CODE REVIEW

**Comment by RWOverdijk** [comment]
@VMBindraban Review please.

File: bin/swan-configure.js [hunk 1]
```diff
@@ -15,11 +15,8 @@ const questions   = {
     message : 'What is your github username:',
     default : (config.git) ? config.git.user.name : '',
     validate: function (input) {
-      if (input && input.length >= 5) {
-        return true;
-      }
-
-      return `Expected a valid github username.`;
+      let valid = typeof input === 'string' && input.length > 0;
+      return valid || `Expected a valid github username.`;
```
Comment (RWOverdijk) [inline bin/swan-configure.js]: Newline before return statement

File: bin/swan-configure.js [hunk 2]
```diff
@@ -15,11 +15,8 @@ const questions   = {
     message : 'What is your github username:',
     default : (config.git) ? config.git.user.name : '',
     validate: function (input) {
-      if (input && input.length >= 5) {
-        return true;
-      }
-
-      return `Expected a valid github username.`;
+      let valid = typeof input === 'string' && input.length > 0;
```
Comment (RWOverdijk) [inline bin/swan-configure.js]: Mixing return types isn't ideal. I'd prefer it the way it was (conditional) but I won't decline the PR if you don't.

File: bin/swan-configure.js (hunk 1 above)
Comment (ppvg) [inline bin/swan-configure.js]: Time for better linting rules? 😃

File: bin/swan-configure.js (hunk 2 above)
Comment (ppvg) [inline bin/swan-configure.js]: Also something a linter might be able to catch (at least in some cases). I get your point, and it's unfortunate that `inquirer` expect different types in the first place.

**Comment by ppvg** [comment]
Different question: how about the ``template literal`` notation where a simple `'string'` would suffice?

**Comment by RWOverdijk** [comment]
@ppvg That should be a simple string then. Probably a refactor that wasn't done properly

**Comment by VMBindraban** [comment]
:+1:

**Pull request merged**